bridge_adaptivity/module/migrations/0005_auto_20180111_1838.py

```diff
@@ -26,11 +26,11 @@ class Migration(migrations.Migration):
             name='engine_name',
             field=models.CharField(blank=True, max_length=255, null=True, unique=True),
         ),
-        migrations.AddField(
-            model_name='engine',
-            name='is_default',
-            field=models.BooleanField(default=False, help_text='If checked Engine will be used as the default!'),
-        ),
+        # migrations.AddField(
```

Comment:
Why have you commented part of migration file? We've discussed that new migrations should be merged into one file. And I would expect its name starts with `0006_`

Reply:
I commented it because after merge there was 2 times adding the same field. that's why I commented this line.

Reply:
this part of migration code should be removed from the file 0005_auto_20180119_1446.py